R/read_secuTrial.R

```diff
@@ -7,7 +7,8 @@
 #'               (ignored when reference values are not in a separate table)
 #' @param dates  logical - convert date variables
 #'
-#' @return secuTrialdata object - a list with one data.frame for each file on the export.
+#' @return secuTrialdata object - a list with one data.frame for each file on the export
```

Comment:
\code{secuTrialdata}

Reply:
Maybe I need to go through all the files and to that everywhere. I'm pretty sure its not consistent.